test/host/xrt/test.cpp

```diff
@@ -1304,7 +1335,7 @@ int start_test(options_t options) {
     auto cclo_ip = xrt::ip(device, xclbin_uuid,
                            "ccl_offload:{ccl_offload_" + cclo_id + "}");
     auto hostctrl_ip = xrt::kernel(device, xclbin_uuid,
-                                   "hostctrl:{hostctrl_" + cclo_id + "_0}",
+                                   "hostctrl:{hostctrl_" + cclo_id + "}",
```

Comment:
The default kernel names were [changed](https://github.com/Xilinx/ACCL/commit/afd0668a10351c76fda76cd0420fe1ec6ba6e465#diff-91c6c2d9e3eec9cefef5e1f7e2a2e4dfb7d41303e6e5afce4c201fac2c4b3350R22) some time ago, so this doesn't run with the default ACCL xclbins.